### ISSUE TITLE
bug: user switch language to "zh_hant" will get 400

### ISSUE THREAD

**Issue by JokerCatz** [uxbox/uxbox#4884]
### Steps To Reproduce

1. user set the language to zh_hant / 繁體中文 and save

### Expected behavior

no err

### Actual behavior

get 400 page

### Desktop (please complete the following information)

self hosted by offical docker-compose

### Frontend Stack Trace

I think you only need this ...  the "lang" value validation limit size to 5 , but "zh_hant" len is 7

```
Data:
{:type :validation, :code :params-validation, :hint "http error"}

Schema: 
[:map
 {:title "update-profile"}
 [:fullname [:app.common.schema/word-string {:max 250}]]
 [:lang {:optional true} [:string {:max 5}]]
 [:theme {:optional true} [:string {:max 250}]]]

Errors:
[{:path [:lang], :in [:lang], :schema [:string {:max 5}], :value "zh_hant"}]

Value:
{:fullname "JokerCatz", :theme "default", :lang "zh_hant"}
```

**Comment by madalenapmelo-kp** [comment]
Hi @JokerCatz,

Thanks for reporting this! We've added this to our backlog on Taiga so that we can look further into it, you can find the details here: https://tree.taiga.io/project/penpot/issue/8469

**Comment by superalex** [comment]
Hello @JokerCatz 

We just released 2.1.2 docker images of Penpot where this issue should be fixed.  Thank you very much!

**Comment by madalenapmelo-kp** [comment]
This issue has been fixed and the fix will be release soon!
